### PR TITLE
[cisco_secure_email_gateway] Fix grok pattern and timezone in consolidated pipeline.

### PR DESCRIPTION
**Pull request by piyush-elastic**
Type of change
- Bug

## What does this PR do?
- Fix grok pattern and Time zone in consolidated pipeline.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## How to test this PR locally
- Clone integrations repo.
- Install elastic package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/cisco_secure_email_gateway directory.
- Run the following command to run tests. 
> elastic-package test

## Related issues

- https://github.com/elastic/sdh-beats/issues/3258 

## Screenshots
Asset test
Pipeline
Static test
System test

### CODE REVIEW

File: packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_consolidated_event.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ processors:
...
...ail_gateway.log.esa.tls.in.cipher} )?(?:ESATLSInConnStatus=%{WORD:cisco_secure_email_gateway.log.esa.tls.in.connection_status} )?(?:ESATLSInProtocol=%{DATA:cisco_secure_email_gateway.log.esa.tls.in.protocol} )?(?:ESATLSOutCipher=%{DATA:cisco_secure_email_gateway.log.esa.tls.out.cipher} )?(?:ESATLSOutConnStatus=%{WORD:cisco_secure_email_gateway.log.esa.tls.out.connection_status} )?(?:ESATLSOutProtocol=%{DATA:cisco_secure_email_gateway.log.esa.tls.out.protocol}($|\\s))?(?:ESADaneHost=%{DATA:cisco_secure_email_gateway.log.esa.dane.host} )?(?:ESADaneStatus=%{WORD:cisco_secure_email_gateway.log.esa.dane.status}($|\\s))?(?:ESADHASource=%{IP:cisco_secure_email_gateway.log.esa.dha_source}($|\\s))?(?:ESADMARCVerdict=%{WORD:cisco_secure_email_gateway.log.esa.dmarc_verdict}($|\\s))?(?:cs5Label=%{DATA:cisco_secure_email_gateway.log.cs5_label}($|\\s))?(?:cs5=%{DATA:cisco_secure_email_gateway.log.cs5}($|\\s))?(?:ESAMARAction=%{DATA:cisco_secure_email_gateway.log.esa.mail_auto_remediation_action}($|\\s))?(?:ESAMsgTooBigFromSender=%{WORD:cisco_secure_email_gateway.log.esa.msg_too_big_from_sender}($|\\s))?(?:ESARateLimitedIP=%{GREEDYDATA:cisco_secure_email_gateway.log.esa.rate_limited_ip}($|\\s))?"
```
Comment (efd6) [inline packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_consolidated_event.yml:9]:
```suggestion
        - ^(?:%{DATA:_tmp.timestamp} )?\
          CEF:%{NUMBER:cisco_secure_email_gateway.log.cef_format_version}\
          \|%{WORD:cisco_secure_email_gateway.log.appliance.vendor}\
          \|%{DATA:cisco_secure_email_gateway.log.appliance.product}\
          \|%{DATA:cisco_secure_email_gateway.log.appliance.version}\
          \|%{DATA:cisco_secure_email_gateway.log.event_class_id}\
          \|%{DATA:cisco_secure_email_gateway.log.event.name}\
          \|%{WORD:event.severity}\
          \|%{GREEDYDATA:_tmp.details} endTime=%{DATA:event.end}%{SPACE}\
          (?:ESADKIMVerdict=%{WORD:cisco_secure_email_gateway.log.esa.dkim_verdict} )?\
          (?:ESADLPVerdict=%{WORD:cisco_secure_email_gateway.log.esa.dlp_verdict} )?\
          (?:dvc=%{IP:cisco_secure_email_gateway.log.data.ip} )?\
          (?:ESAAttachmentDetails=%{DATA:cisco_secure_email_gateway.log.esa.attachment_details} )?\
          (?:ESAFriendlyFrom=%{DATA:cisco_secure_email_gateway.log.esa.friendly_from} )?\
          (?:ESAGMVerdict=%{WORD:cisco_secure_email_gateway.log.esa.graymail_verdict} )?\
          startTime=%{DATA:event.start}%{SPACE}\
          (?:(deviceInboundInterface||deviceOutboundInterface)=%{WORD:cisco_secure_email_gateway.log.listener.name} )?\
          (?:deviceDirection=%{DATA:cisco_secure_email_gateway.log.device_direction} )?\
          (?:ESAMailFlowPolicy=%{WORD:cisco_secure_email_gateway.log.esa.mail_flow_policy} )?\
          (?:suser=%{DATA:email.from.address} )?\
          (?:cs1Label=%{WORD:cisco_secure_email_gateway.log.cs1_label}%{SPACE}\
          cs1=%{WORD:cisco_secure_email_gateway.log.cs1} )?\
          (?:cs2Label=%{WORD:cisco_secure_email_gateway.log.cs2_label}%{SPACE}\
          cs2=%{DATA:cisco_secure_email_gateway.log.cs2} )?\
          (?:ESAMFVerdict=%{WORD:cisco_secure_email_gateway.log.esa.mf_verdict} )?\
          (?:act=%{WORD:cisco_secure_email_gateway.log.act} )?\
          (?:ESAFinalActionDetails=%{DATA:cisco_secure_email_gateway.log.esa.final_action_details} )?\
          (?:cs4Label=%{DATA:cisco_secure_email_gateway.log.cs4_label} )?\
          (?:cs4='<%{DATA:cisco_secure_email_gateway.log.cs4}>' )?\
          (?:ESAMsgSize=%{NUMBER:cisco_secure_email_gateway.log.esa.msg_size:long} )?\
          (?:ESAOFVerdict=%{WORD:cisco_secure_email_gateway.log.esa.outbreak_filter_verdict} )?\
          (?:duser=%{DATA:email.to.address} )?\
          (?:ESAHeloDomain=%{DATA:cisco_secure_email_gateway.log.esa.helo.domain} )?\
          (?:ESAHeloIP=%{IP:cisco_secure_email_gateway.log.esa.helo.ip} )?\
          (?:ESAReplyTo=%{DATA:cisco_secure_email_gateway.log.esa.reply_to} )?\
          (?:cfp1Label=%{WORD:cisco_secure_email_gateway.log.cfp1_label}%{SPACE}\
          cfp1=%{DATA:_tmp.cfp1} )?\
          (?:ESASDRDomainAge=%{DATA:cisco_secure_email_gateway.log.esa.sdr_consolidated_domain_age} )?\
          (?:cs3Label=%{WORD:cisco_secure_email_gateway.log.cs3_label} )?\
          (?:cs3=%{DATA:cisco_secure_email_gateway.log.cs3} )?\
          (?:cs6Label=%{DATA:cisco_secure_email_gateway.log.cs6_label} )?\
          (?:cs6=%{DATA:cisco_secure_email_gateway.log.cs6} )?\
          (?:ESASPFVerdict=%{DATA:cisco_secure_email_gateway.log.esa.spf_verdict} )?\
          (?:sourceHostName=%{DATA:source.domain} )?\
          (?:ESASenderGroup=%{DATA:cisco_secure_email_gateway.log.esa.sender_group} )?\
          (?:sourceAddress=%{IP:source.ip} )?\
          msg=('|")%{DATA:email.subject}('|")($|\s)\
          (?:ESAURLDetails=%{DATA:cisco_secure_email_gateway.log.esa.url_details}($|\s))?\
          (?:ESATLSInCipher=%{DATA:cisco_secure_email_gateway.log.esa.tls.in.cipher} )?\
          (?:ESATLSInConnStatus=%{WORD:cisco_secure_email_gateway.log.esa.tls.in.connection_status} )?\
          (?:ESATLSInProtocol=%{DATA:cisco_secure_email_gateway.log.esa.tls.in.protocol} )?\
          (?:ESATLSOutCipher=%{DATA:cisco_secure_email_gateway.log.esa.tls.out.cipher} )?\
          (?:ESATLSOutConnStatus=%{WORD:cisco_secure_email_gateway.log.esa.tls.out.connection_status} )?\
          (?:ESATLSOutProtocol=%{DATA:cisco_secure_email_gateway.log.esa.tls.out.protocol}($|\s))?\
          (?:ESADaneHost=%{DATA:cisco_secure_email_gateway.log.esa.dane.host} )?\
          (?:ESADaneStatus=%{WORD:cisco_secure_email_gateway.log.esa.dane.status}($|\s))?\
          (?:ESADHASource=%{IP:cisco_secure_email_gateway.log.esa.dha_source}($|\s))?\
          (?:ESADMARCVerdict=%{WORD:cisco_secure_email_gateway.log.esa.dmarc_verdict}($|\s))?\
          (?:cs5Label=%{DATA:cisco_secure_email_gateway.log.cs5_label}($|\s))?\
          (?:cs5=%{DATA:cisco_secure_email_gateway.log.cs5}($|\s))?\
          (?:ESAMARAction=%{DATA:cisco_secure_email_gateway.log.esa.mail_auto_remediation_action}($|\s))?\
          (?:ESAMsgTooBigFromSender=%{WORD:cisco_secure_email_gateway.log.esa.msg_too_big_from_sender}($|\s))?\
          (?:ESARateLimitedIP=%{GREEDYDATA:cisco_secure_email_gateway.log.esa.rate_limited_ip}($|\s))?
        - ^%{GREEDYDATA:cisco_secure_email_gateway.log.message}$
```
(more lines, but greater clarity in diffs in the future — also no `\\` doubling)

Comment (piyush-elastic) [reply packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_consolidated_event.yml:9]: @efd6 , Agree that we cans split it in multiple lines , but we can't enclose the whole GROK in double quotes as we have some match condition specifically for single quotes within GROK so, will update in multiple lines only.

Comment (efd6) [reply packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_consolidated_event.yml:9]: It doesn't need quotes. The patterns that I posted are tested working.

Comment (efd6) [reply packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_consolidated_event.yml:9]: Followed up offline.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**